packages/cognito-auth/src/utils/cognito.ts

```diff
@@ -113,6 +113,7 @@ export const deserializeIdToken = (loginSession: Partial<LoginSession> | undefin
     clientId: decoded['custom:reapit:clientCode'] || null,
     adminId: decoded['custom:reapit:marketAdmin'] || null,
     userCode: decoded['custom:reapit:userCode'] || null,
+    isAdmin: Boolean(decoded['cognito:groups']?.includes('CustomerAdministrators')),
```

Comment:
I added 

> One small thing, because you are adding this, can you also quickly look at this ticket? #1262 It's for a developer so am sure they will appreciate!

I added userTel field. You can review it again

Reply:
Thanks for doing this!